.github/workflows/verify.yml

```diff
@@ -220,6 +220,7 @@ jobs:
       uses: actions/download-artifact@v4.1.7
       with:
         name: xnvme-src-archive-with-subprojects
+
```

Comment:
This re-format, could it be moved into it's own commit? Or is it essential for the feature:
```
feat(ci/build): run test-ramdisk in build-windows workflow 
```
?

Reply:
Not essential, i have moved it to another commit now :)